tests/unit/test_main.py

```diff
@@ -1289,11 +1292,11 @@ def test_multiple_configs(capsys, tmpdir):
     main.main([str(tmpdir), "--resolve-all-configs", "--cr", str(tmpdir), "--verbose"])
     out, _ = capsys.readouterr()
 
-    assert f"{str(setup_cfg_file)} used for file {str(file1)}" in out
-    assert f"{str(pyproject_toml_file)} used for file {str(file2)}" in out
-    assert f"{str(isort_cfg_file)} used for file {str(file3)}" in out
-    assert f"default used for file {str(file4)}" in out
-    assert f"default used for file {str(file5)}" in out
+    assert f"{setup_cfg_file!s} used for file {file1!s}" in out
+    assert f"{pyproject_toml_file!s} used for file {file2!s}" in out
+    assert f"{isort_cfg_file!s} used for file {file3!s}" in out
+    assert f"default used for file {file4!s}" in out
+    assert f"default used for file {file5!s}" in out
```

Comment:
The conversion to `!s` was semantically equivalent but unnecessary here, as far as I can tell.

```suggestion
    assert f"{setup_cfg_file} used for file {file1}" in out
    assert f"{pyproject_toml_file} used for file {file2}" in out
    assert f"{isort_cfg_file} used for file {file3}" in out
    assert f"default used for file {file4}" in out
    assert f"default used for file {file5}" in out
```

(Objects' `.__str__()` methods are called automatically.)